lib/companies_web/router.ex

```diff
@@ -35,19 +43,17 @@ defmodule CompaniesWeb.Router do
       resources "/companies", CompanyController, except: [:index, :show]
       resources "/jobs", JobController, except: [:index, :show]
     end
+
+    scope "/admin", Admin do
```

Comment:
I don't know that this will ever be translated, does it need to live under the locale?

Reply:
We have navigation links to translated pages in the admin panel. For their work, we need to pass the current locale. For such cases, it is easier to that all pages are under the locale.

Reply:
In the current implementation using locale helper and the default locale, we can move the admin panel from under the locale, but the locale will be lost when going through the admin panel.
I think it's better to leave it under the locale.